### PR TITLE
feat: add addition info for consul

### PR DESCRIPTION
**Pull request by L2ncE**
#### What type of PR is this?

feat

#### What this PR does / why we need it (English/Chinese):

在之前的 Consul 拓展实现中，直接使用 Hertz 提供的 Info 结构体中的 Tags 作为 Consul 的 Meta，并且 Consul 非常重要的 Tags 也没有进行配置的入口。因此在这里提供了一个新的结构体来为用户方便配置 Tags 与 Meta。

In the previous Consul extension implementation, the Tags field in the Info structure provided by Hertz was directly used as the Meta value of Consul, and the very important Tags field of Consul did not have an entry for configuration.
Therefore, a new structure is provided here to facilitate the configuration of Tags and Meta for users.

#### Which issue(s) this PR fixes:

https://github.com/cloudwego/hertz/issues/596

### CODE REVIEW

**Comment by welkeyever** [comment]
Please check the Tests / ut - there are issues in it.

**Comment by L2ncE** [comment]
> Please check the Tests / ut - there are issues in it.

It seems to be a Nacos problem, not caused by this PR. Can you rerun it for me, and I will check if there is still a problem.

**Comment by GuangmingLuo** [comment]
@L2ncE please check the test result.

**Comment by L2ncE** [comment]
> @L2ncE please check the test result.

Got it, I'll fix it in my spare time.

**Comment by L2ncE** [comment]
@welkeyever @GuangmingLuo PTAL❤️

**Comment by GuangmingLuo** [comment]
@li-jin-gou please take a look

**Review by welkeyever — APPROVED** [review]
LGTM, thx~

File: consul/registry.go [hunk 1]
```diff
@@ -88,7 +99,8 @@ func (c *consulRegistry) Register(info *registry.Info) error {
 		Name:    info.ServiceName,
 		Address: host,
 		Port:    port,
-		Meta:    info.Tags,
```
Comment (li-jin-gou) [inline consul/registry.go:91]: 这个地方有问题

Comment (L2ncE) [reply consul/registry.go:91]: > 这个地方有问题

什么问题

**Comment by li-jin-gou** [comment]
齐哥在跟进修复了



---Original---
From: "Xinhao ***@***.***&gt;
Date: Thu, Dec 28, 2023 11:48 AM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [hertz-contrib/registry] feat: add addition info for consul (PR#86)




 
@L2ncE commented on this pull request.
 
 
In consul/registry.go:
 &gt; @@ -88,7 +99,8 @@ func (c *consulRegistry) Register(info *registry.Info) error {  		Name:    info.ServiceName,  		Address: host,  		Port:    port, -		Meta:    info.Tags,   
这个地方有问题
  
什么问题

**Pull request merged**